File: ExampleSwift/DetailVC.m
//
//  DetailVC.m
//  ExampleSwift
//
//  Created by 刘波 on 2021/5/11.
//

#import "DetailVC.h"
#import "ExampleSwift-Swift.h"

@implementation DetailVC

- (void)viewDidLoad {
    [super viewDidLoad];
    self.view.backgroundColor = UIColor.systemGreenColor;
}


- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    ViewController *vc = [[ViewController alloc] init];
    [self presentViewController:vc animated:YES completion:nil];
}

@end

File: Example/DetailVC.swift
//
//  DetailVC.swift
//  Example
//
//  Created by 刘波 on 2021/5/11.
//

import UIKit

class DetailVC: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        view.backgroundColor = .systemGreen
    }
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let vc = ViewController()
        present(vc, animated: true, completion: nil)
    }
}
